chapters/VK_EXT_debug_report.txt

```diff
@@ -168,7 +168,7 @@ type of object handle being reported, are:
 include::{generated}/api/enums/VkDebugReportObjectTypeEXT.txt[]
 
 [[debug-report-object-types]]
-.VkDebugReportObjectTypeEXT and Vulkan Handle Relationship
+.etext:VkDebugReportObjectTypeEXT and Vulkan Handle Relationship
```

Comment:
This and a couple of similar changes make the 'allchecks' target fail, which breaks our internal CI. The problem is that etext: isn't just a "format this like elink, but don't link" macro, it's semantically actually different (probably should be called enum:). The checker script detects that etext: is being used with something which isn't an enumerant name. For the time being I'm reverting these changes despite the minor formatting improvements they would otherwise offer.

Reply:
@oddhack Hm, the styleguide says `etext` is not validated.

So, what do I need to do with this PR? Did you revert the whole thing, or just the bad occurences internally?